submissions/jonasMwansa/css/html/landing-page.html

```diff
@@ -37,72 +37,73 @@ <h1>FoodBag</h1>
     </div>
 
     <section>
-            <form action="#" method="post" enctype="multipart/form-data">
-                <label for="location">
-                    <b>Where are you?</b>
-                </label>
+            <form id="locationForm" method="post" enctype="multipart/form-data">
+                <label for="location"><b>Where are you?</b></label>
                 <br>
                     <select name="address" id="location">
-                        <option value="bodjila market">Bodija market</option>
-                        <option value="Nkwashi AIR park">Nkwashi, AIR park</option>
-                        <option value="Nkwashi prefab 01">Nkwashi, prefab 01</option>
-                        <option value="Nkwashi AIR Main house 01">Nkwashi, AIR Main house 01</option>
-                        <option value="Nkwashi AIR Main house 03">Nkwashi, AIR Main house 03</option>
-                        <option value="Nkwashi prefab 02">Nkwashi, prefab 02</option>
-                        <option value="Nkwashi Main gate">Nkwashi, Main gate</option>
-                        <option value="Nkwashi Main house">Nkwashi, Main house</option>
+                        <option value disabled selected>select location</option>
```

Comment:
value disabled selector should be joined together i.e value_disabled_selector. Ensure to effect this changes in your css and js files